x/crisis/go.mod

```diff
@@ -0,0 +1,156 @@
+module cosmossdk.io/x/crisis
+
+go 1.19
+
+replace (
+	cosmossdk.io/store => ../../store
+	github.com/cosmos/cosmos-sdk => ../..
+
+	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
+	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+)
```

Comment:
can you add these replace imports at the end of the file to be consistent with other go.mod's ?